compsat/ls-ganap
# Feature/add org users

SUMMARY:

- Updated the models for OrgHost and SangguHost to contain a user field.
- Updated the scraper for org users to match the necessary changes made
- Updated the fixture for organizations.yaml to include the user field
- Created the fixture for org_users.yaml
- Created the fixture for sanggu.yaml
- Created the fixture for sanggu_users.yaml

THINGS TO FIX:

- The Sanggu entries still do not have a proper description included on them
- The offices still do not have proper fixtures for both the OfficeHost and office users

TEST PLAN:
- Checkout the branch feature/add_org_users
- Rerun the migrations
- Run docker-compose up and see if the fixtures ran properly
